ear/src/main/resources/s3binaries/nuxeo.defaults

```diff
@@ -12,3 +13,10 @@ nuxeo.s3storage.crypt.key.alias=
 nuxeo.s3storage.crypt.key.password=
 
 nuxeo.core.binarymanager=org.nuxeo.ecm.core.storage.sql.S3BinaryManager
+
+nuxeo.s3storage.useDirectUpload=true
+
+nuxeo.s3storage.direct.bucket=
```

Comment:
"direct" is not very meaningful on its own. I'd like a name that's about the role of the bucket itself in the system, not the software component that's using it. Maybe `nuxeo.s3storage.transient.bucket`?